draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -607,28 +559,32 @@ Note that sometimes these thresholds are configurable and sometime they are hard
+[Editorial Note: This section in under review.]
 
-Note that these vendor-specific attributes could be 
+In certain cases, a coherent pluggable may support attributes that are specific to a particular vendor. This draft refers to such attributes as "Opaque Attributes". Given that coherent pluggables encompass capability, configuration, and performance monitoring (PM)/state attributes, each category may contain opaque attributes. Consequently, the opaque attributes could include the following:
 
-* read-only capability attributes
-* read-write configuration attributes or 
-* read-only performance monitoring attributes. 
+* Read-only opaque capability attributes
+* Read-write opaque configuration attributes
+* Read-only opaque PM/state attributes
 
-As part of coherent pluggable work, we need to address this situation where a vendor has proprietary attributes which need to be configured on coherent pluggables or read from them. In this situation we also need to address how the proprietary attributes are treated by both host and internal protocol between host and pluggable (i.e., CMIS protocol). This allows different coherent pluggables to be used in various multi-vendor hosts (e.g., routers) in plug-and-play fashion.
+As part of coherent pluggable work, we need to address this situation where a coherent pluggable contains some proprietary capability, configuration and PM/states attributes which are needed to be configured or accessed from coherent pluggables. In this situation we need to address how opaque attributes are treated by packet device host. This allows different coherent pluggables to be used in various multi-vendor hosts in plug-and-play fashion.
 
-To achieve this, the coherent pluggable YANG data model (the work done as part of this draft -  Google Sheet) should first be augmented with vendor proprietary capability, configuration or PM attributes. As noted above, it might be also necessary to define how these new attributes are mapped to the internal protocol between the host and the pluggable via CMIS protocol. A key consideration is that the host does not need to understand the semantics of these new attributes and may not even need to know their syntax.
+It is important to note that "opaque attributes" are not simply attributes that can be addressed through augmentation of the YANG data model. The reason for this is that the coherent pluggable is exposed to external systems via the host packet device northbound interface as shown in {{figure-details-packet-optical-device}}. If the host packet device does not recognize any of these "opaque attributes". it may prevent the discovery of the coherent pluggable.
```

Comment:
I don't get this rationale. it is up to the host implementation to decide which attributes to recognize and support. Even non-opaque attributes may not be supported on all platforms and for good reasons.